apps/gp2-server/src/app.ts

```diff
@@ -80,6 +91,8 @@ export const appFactory = (libs: Libs = {}): Express => {
    * Public routes --->
    */
   const userController = libs.userController || new Users(userDataProvider);
+  const workingGroupController =
+    libs.workingGroupController || new WorkingGroup(workingGroupDataProvider);
```

Comment:
The name should be plural to be consistent with other controllers. Same for the filename, and the data provider.

Reply:
so the name of the class should be in plural to be consistent with others but the file name and data provider should not, as seen in the example above. It might be inconsistent with CRN maybe? Either way i think we should sort out the naming strategies in this repo

Reply:
Agreed, let's create a ticket.